unit-tests/CMakeLists.txt

```diff
@@ -41,8 +41,8 @@ foreach(_test ${UNIT_TEST_LIST})
     ${PFUNIT_DIR}/lib/libpfunit.a 
     gsilib_global${debug_suffix} gsilib_shrd${debug_suffix} gsilib_global${debug_suffix} gsilib_shrd${debug_suffix} 
          gsilib_global${debug_suffix} ${FRAMEPACK} ${FRAMEMODULE}
-         ${NETCDF_LIBRARIES_F90} ${NETCDF_LIBRARIES} ${HDF5_HL_LIBRARIES} ${MPI_Fortran_LIBRARIES}   
-         ${LAPACK_LIBRARIES} -L./ ${EXTRA_LINKER_FLAGS} ${HDF5_LIBRARIES} ${CURL_LIBRARIES} ${CORE_LIBRARIES} 
+         ${NETCDF_LIBRARIES_F90} ${NETCDF_LIBRARIES} ${MPI_Fortran_LIBRARIES}   
```

Comment:
Why the discrepancy in NETCDF_LIBRARIES and NETCDF_LIBRARIES_F90?

Reply:
Found that unit-tests do not use NetCDF so remove NetCDF from CMakeLists.txt